config/environments/production.rb

```diff
@@ -28,7 +38,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
@s-espinosa @icorson3 I see a lot of conflicting information about setting this to true or false